target/bin/sedfile

```diff
@@ -11,36 +11,34 @@
 
 set -ueo pipefail
 
-HASHTOOL="sha1sum"
-SKIP_ERROR=0
-
-function usage { echo "Usage: ${0} -i <replace/delete operation> <file>" ; }
+HASHTOOL='sha1sum'
+SKIP_ERROR=false
 
 if [[ $# -lt 3 ]]
 then
-  usage
-  _log 'error' 'At least three parameters must be given'
+  echo 'At least three parameters must be given'
+  echo -e "Usage: ${0} -i <replace/delete operation> <file>\n"
   exit 1
 fi >&2
 
-[[ -f /CONTAINER_START ]] && SKIP_ERROR=1 # Hide error, if container was restarted.
-if [[ "${1}" == "--strict" ]]         # Show error every time.
+[[ -f /CONTAINER_START ]] && SKIP_ERROR=true # hide error if container was restarted
+if [[ ${1} == '--strict' ]]                  # show error every time
 then
-  SKIP_ERROR=0
+  SKIP_ERROR=false
   shift
 fi
 
 # get last argument
 FILE=${*: -1}
 
-OLD=$(${HASHTOOL} "${FILE}")
-sed "$@"
-NEW=$(${HASHTOOL} "${FILE}")
+OLD="$(${HASHTOOL} "${FILE}")"
+sed "${@}"
+NEW="$(${HASHTOOL} "${FILE}")"
```

Comment:
When assigning a value to a variable and using `$()`, no quoting is required.
```suggestion
OLD=$(${HASHTOOL} "${FILE}")
sed "${@}"
NEW=$(${HASHTOOL} "${FILE}")
```

Reply:
👍🏼

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7